website/docs/reference/datasources/authenticated-api.md

```diff
@@ -0,0 +1,71 @@
+---
+description: Connect Appsmith to an Authenticated API and create queries.
+---
+
+# Authenticated API
+
+This page gives information to connect Appsmith to an Authenticated API and to read and write data in your applications.
+
+Use this datasource to create multiple queries for the same API. Every query created from this datasource has shared configuration (root URL, authentication, headers, and so on) to avoid re-entering details. If you're only creating a single query for your API, try using a [REST API](/reference/datasources/rest-api) datasource.
```

Comment:
This was a point brought up by Balaji where users very frequently are confused about whether they should be using the Auth API or REST API datasources, and requested that we add something to help users decide on which is best for their situation.

I put it up near the top so that a user has the chance to understand what this datasource is used for, and whether they're on the right page for what they want to accomplish